migrations/1575287807413-remove_create_permissions_thuringia.js

```diff
@@ -0,0 +1,110 @@
+// eslint-disable-next-line no-unused-vars
+const { info, error } = require('../src/logger');
+
+const { connect, close } = require('../src/utils/database');
+
+const Role = require('../src/services/role/model');
```

Comment:
Please re-define the model to avoid errors when multiple migrations are run in one go